mandatory/1-fix-functions.js

```diff
@@ -21,9 +20,9 @@ function mood() {
 }
 
 function greaterThan10(num) {
-  let isBigEnough;
+  let isBigEnough = num;
 
-  if (isBigEnough) {
+  if (isBigEnough >10) {
     return "num is greater than 10";
   } else {
     return "num is not big enough";
```

Comment:
line 23 is redundant at the moment, you could do `let isBigEnough = num > 10` here and then you can use it directly in your if statement